bike-erp/backend/src/main/helpers/db.ts

```diff
@@ -0,0 +1,15 @@
+import mysql from 'mysql2';
+
+const db = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: 'mysqlpw',
```

Comment:
could leave the password empty such that we can input our own pwd

Reply:
Done!